bridge/svix-bridge-plugin-queue/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 
 [dependencies]
 futures-lite = "1.12.0"
-generic_queue = { path = "../generic-queue" }
+omniqueue = { path = "/home/onelson/Projects/omniqueue-rs/omniqueue" }
```

Comment:
I'm working through some stuff 😓 
Once I know what I need, I'll put up a PR on the omniqueue side.